key_value_store.cpp

```diff
@@ -0,0 +1,20 @@
+#include "key_value_store.h"
+
+void KeyValueStore::set(const std::vector<char>& key, const FlaggedValue& fv) {
+  std::lock_guard<std::mutex> lk(mut);
```

Comment:
Eh, it's a short function, so not really necessary, but ```lk -> lock```. This is really just being pedantic.